rustfmt.toml

```diff
@@ -1,2 +1,2 @@
 write_mode = "Overwrite"
-
+closure_block_indent_threshold = 1
```

Comment:
The rustfmt.toml changes were there to prevent even more of this, although I'm not sure why this particular case still occurred.